src/terminal/input/lib/terminalinput.vcxproj

```diff
@@ -6,10 +6,12 @@
     <RootNamespace>adapter</RootNamespace>
     <ProjectName>TerminalInput</ProjectName>
     <TargetName>TerminalInput</TargetName>
-    <ConfigurationType>StaticLibrary</ConfigurationType>
+    <ConfigurationType>StaticLibrary</ConfigurationType>
   </PropertyGroup>
   <Import Project="$(SolutionDir)src\common.build.pre.props" />
   <ItemGroup>
+    <ClCompile Include="..\mouseInput.cpp" />
+    <ClCompile Include="..\mouseInputState.cpp" />

```

Comment:
you've updated a vcxproj file: you have to update the `sources` file in the same directory

Reply:
(Or the `sources.inc` file in the directory above it)